.circleci/config.yml

```diff
@@ -69,6 +69,31 @@ node-bench-e2e-base: &node-bench-e2e-base
         name: E2E Benchmark
         command: yarn bench:e2e
 
+node-bench-sirun-base: &node-bench-sirun-base
+  resource_class: small
+  working_directory: ~/dd-trace-js
+  steps:
+    - checkout
+    - *yarn-versions
+    - *restore-yarn-cache
+    - *yarn-install
+    - *save-yarn-cache
+    - run:
+        name: Install system deps
+        command: apt-get update && apt-get install -y valgrind wget
+    - run:
+        name: Install sirun
+        command: wget https://github.com/DataDog/sirun/releases/download/v0.1.3/sirun-v0.1.3-x86_64-unknown-linux-gnu.tar.gz && tar zxf sirun-v0.1.3-x86_64-unknown-linux-gnu.tar.gz && mv sirun /usr/bin
+    - run:
+        name: Run sirun
+        command: |
+          cd $SIRUN_TEST_DIR
+          node /root/dd-trace-js/benchmark/sirun/run-all-variants.js | tee /tmp/sirun-output.ndjson
```

Comment:
will this also write any stdout from the script to the file?

Reply:
stdout and stderr from tested programs are suppressed in this PR with `SIRUN_NO_STDIO=1`.

Reply:
ah, thanks, I knew I was missing something